document/FileFormat.mdk

```diff
@@ -2696,31 +2703,318 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | hexadecimal                 | unconditional.              |                             |
+|                  | string representation of    |                             |                             |
+|                  | that address or offset.     |                             |                             |
+|                  | Absent: If the target of    |                             |                             | 
+|                  | the branch is not           |                             |                             |
+|                  | known.|                     |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `call`           | Recommended: The            | Optional: An array of       | Optional: The names and     |
+|                  | fully qualified name of     | strings representing        | values of any variables     |
+|                  | the function being          | the arguments               | in  the expressions passed  |
+|                  | called.                     | passed to the               | as the arguments to the     |  
+|                  |                             | function.                   | function.                   |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `callReturn`     | Recommended: The            | Optional: If the function   | Absent                      |
+|                  | fully qualified name of     | returns a value or          |                             |
+|                  | the  function being         | values, in which case,      |                             |
+|                  | returned from.              | string representations      |                             |
+|                  |                             | of the value or values      |                             |
+|                  |                             | returned.                   |                             |
+|                  |                             | Absent: If the function     |                             |
+|                  |                             | does not return a value.    |                             |
+|                  |                             |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `continuation`   | Absent                      | Absent                      | Optional: The names and     |
+|                  |                             |                             | values of selected variables|
+|                  |                             |                             | at the specified location.  |
```

Comment:
>at the specified location [](start = 81, length = 25)

'at the specified location' is a bit strange here, as selected variables could be elsewhere (you might refer to the name/value of a global static variable, for example). Can we clarify? We want to clearly indicate these name/value pairs represent current state when the specified location was executing. We don't want to suggest we only permit documenting variables specified at the current location.